app/models/user.rb

```diff
@@ -10,7 +51,9 @@ class User < ApplicationRecord
   has_many :org_edits, through: :permissions, source: :organization
   has_many :prog_edits, through: :permissions, source: :program
 
+  accepts_nested_attributes_for :media, allow_destroy: true
+
   # Validations
-  validates :first_name, :last_name, :linkedin_id, :contact_url, :role, presence: true
-  validates :visible, inclusion: { in: [true, false] }
+  validates :first_name, :last_name, :contact_url, presence: true
+  # validates :visible, inclusion: { in: [true, false] }
```

Comment:
Was this validation breaking something?

Reply:
yes, because role and visibility are being set after initialization

Reply:
Interesting. I have the same default callback for after_initialize in my branch and my validations are working just fine..